src/main/scala/tech/cryptonomic/conseil/tezos/TezosPlatformDiscoveryOperations.scala

```diff
@@ -295,6 +297,7 @@ class TezosPlatformDiscoveryOperations(
             } else {
               (for {
                 _ <- caching.putAttributes(tableName, attributes)
+                _ <- IO.contextShift(executionContext).shift
```

Comment:
Should switching to IO be put in this pr, or did it directly help with the caching task?

Reply:
that was a change requested by @ivanopagano in previous caching PR, but it was merged before I could push it, so I thought I'll add it here